tests/test_server.py

```diff
@@ -34,13 +35,15 @@ def mocked_create(self, dbservice, collection, data):
 
     def mocked_init(self):
         self.submission_db_service = mock.Mock()
+        self.schema_db_service = mock.Mock()
+        self.backup_db_service = mock.Mock()
 
     @unittest_run_loop
-    @mock.patch("metadata_backend.db_services.CRUDService.create")
-    @mock.patch("metadata_backend.views.SiteHandler.__init__")
-    async def test_submit_works_correct_schema_valid_input(self,
-                                                           mocked_mongodb_serv,
-                                                           mocked_crudservice):
+    @mock.patch("api.views.SiteHandler.__init__")
+    @mock.patch("database.db_services.CRUDService.create")
+    # TODO: Fix this test to use mongomock
```

Comment:
would be nice to track to do in issues instead of code :)

Reply:
true, I'll convert these to issues!